src/status_im/contexts/communities/events.cljs

```diff
@@ -399,22 +399,24 @@
      [:json-rpc/call :schema.common/rpc-call]]]])
 
 (rf/reg-event-fx :communities/get-revealed-accounts-success
- (fn [{:keys [db]} [community-id on-success revealed-accounts-js]]
+ (fn [{:keys [db]} [community-id on-success latest-request-to-join-js]]
    (when-let [community (get-in db [:communities community-id])]
-     (let [revealed-accounts
-           (reduce
-            (fn [acc {:keys [address] :as revealed-account}]
-              (assoc acc address revealed-account))
-            {}
-            (data-store.communities/<-revealed-accounts-rpc revealed-accounts-js))
-
+     (let [request-to-join (js->clj latest-request-to-join-js :keywordize-keys true)
```

Comment:
If you remove `:js-response true` from the event `:communities/get-revealed-accounts`, then the effect `:json-rpc/call` will automatically convert for you the JS response to CLJS and will keywordize keys too.

Reply:
Thanks, great point, made the change.